Update Heli Tuning Wiki (#4104)
ArduPilot/ardupilot_wiki: copter/source/docs/traditional-helicopter-manual-tuning.rst

.. _traditional-helicopter-manual-tuning:

> ===============================

Reviewer: heading over/under line

Manual Tuning Instructions
===============================

Pitch and Roll Axes
===================

> Setting _FF and ACCEL_x_MAX Parameters for Desired Pitch and Roll Response

Reviewer: back ticks to highlite

--------------------------------------------------------------------------

In both pitch and roll axes, the FF gain is set so that the actual aircraft
rate matches the desired rate. To do this, the RATE message in the log is
required to compare the RATE.PDes and RATE.P log data for pitch and the RATE.RDes and RATE.R
log data for roll.

With the FF gains set to 0.15, takeoff and establish a hover
in Stabilize flight mode, then make some sharp stick inputs in both pitch and
roll. Land and pull the log from the microSD card and look at the signals in

> your ground station software. If the actual rate is more than the desired rate

Reviewer 2: ```suggestion
your log reviewing software. If the actual rate is more than the desired rate
```

Author: done. thanks!

properly leveled in the setup. In this case, just make sure the change in rate is similar in 
amplitude between desired and actual.  If you get the rates to match and they feel like they 
are too fast, then reduce the ``ATC_ACCEL_x_MAX`` parameter and repeat the process above to 
match the desired and actual rates. 

If while tuning the FF gain the aircraft starts to oscillate, reduce the 
ATC_ANG_xxx_P gain for that axis until the oscillations stop.  However for most 
helicopters the suggested values above should not cause this problem.

Tuning the D and P gain
-----------------------

Once you have the heli responding nicely with the rate FF gain, now tune the
PID gains. The rate PID controller provides stability to reject disturbances and
keep the actual aircraft following the software demanded rates.
 
Start with the D gain.  Use the :ref:`Transmitter based tuning<common-transmitter-tuning>` feature of ArduCopter.  Make the following parameter changes:

+--------------------------------------------+---------+
| :ref:`TUNE<TUNE>`                          | 21      |
+--------------------------------------------+---------+

> | ``TUNE_LOW`` (prior to 4.0)                | 0       |

Reviewer: we can probably drop the 4.0 stuff now that we are 4.2, and in all the tables below

+--------------------------------------------+---------+
| :ref:`TUNE_MIN<TUNE_MIN>` (4.0 and later)  | 0       |
+--------------------------------------------+---------+
| ``TUNE_HIGH`` (prior to 4.0)               | 30      |
+--------------------------------------------+---------+
| :ref:`TUNE_MAX<TUNE_MAX>` (4.0 and later)  | 0.030*  |
+--------------------------------------------+---------+

*for futaba radios this equates to one increment in the knob to 0.001*

Adjust the tuning knob until the :ref:`ATC_RAT_RLL_D<ATC_RAT_RLL_D__AC_AttitudeControl_Heli>` and :ref:`ATC_RAT_PIT_D<ATC_RAT_PIT_D__AC_AttitudeControl_Heli>` gains are
0.001. Lift into a hover and make some sharp stick inputs in roll.  Most
helicopters will see roll oscillations before they see pitch oscillations.
That is why roll inputs are suggested.  If it doesn't shake, increase the gain

> by 0.001 and try it again. At the value where you get the rapid shaking, cut

Reviewer 2: ```suggestion
by 0.001 and try it again. At the value where you get rapid shaking, cut
```

Author: done

pitch and roll to make sure it's stable.

Now tune the P gains.  Make the following tuning parameter changes:

+--------------------------------------------+---------+
| :ref:`TUNE<TUNE>`                          | 4       |
+--------------------------------------------+---------+
| ``TUNE_LOW`` (prior to 4.0)                | 0       |
+--------------------------------------------+---------+
| :ref:`TUNE_MIN<TUNE_MIN>` (4.0 and later)  | 0       |
+--------------------------------------------+---------+
| ``TUNE_HIGH`` (prior to 4.0)               | 300     |
+--------------------------------------------+---------+
| :ref:`TUNE_MAX<TUNE_MAX>` (4.0 and later)  | 0.3*    |
+--------------------------------------------+---------+

*for futaba radios this equates to one increment in the knob to 0.01*

Adjust the tuning knob until the :ref:`ATC_RAT_RLL_P<ATC_RAT_RLL_P__AC_AttitudeControl_Heli>` and :ref:`ATC_RAT_PIT_P<ATC_RAT_PIT_P__AC_AttitudeControl_Heli>`  gains are
0.05. Lift into a hover and roll aggressively from side to side.  If it doesn't
shake, increase the gain by 0.01 and try it again. At the value where you get

> the rapid shaking, cut that value in half and enter it as the final tuning value

Reviewer 2: ```suggestion
rapid shaking, cut that value in half and enter it as the final tuning value
```

Author: done. thanks
